src/Grid/TablePaginationActions.js

```diff
@@ -0,0 +1,163 @@
+import FirstPageIcon from 'material-ui-icons/FirstPage';
+import IconButton from 'material-ui/IconButton';
+import KeyboardArrowLeft from 'material-ui-icons/KeyboardArrowLeft';
+import KeyboardArrowRight from 'material-ui-icons/KeyboardArrowRight';
+import LastPageIcon from 'material-ui-icons/LastPage';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withStyles } from 'material-ui/styles';
+
+const styles = theme => ({
+  root: {
+    flexShrink: 0,
+    marginLeft: theme.spacing.unit * 2.5
+  },
+  buttonStyle: {
+    height: '30px',
+    width: '30px'
+  }
+});
+
+class TablePaginationActions extends React.Component {
+  state = {
+    pages: [0, 1, 2, 3, 4],
+    activePage: 0
+  }
+
+  handleFirstPageButtonClick = event => {
+    this.props.onChangePage(event, 0);
+
+    this.setCurrentPage(0);
```

Comment:
The `setCurrentPage` maybe should call `this.props.onChangePage` to reduce duplicated code.